### PR TITLE
libreoffice: rebuild on x86

### PR DESCRIPTION
**Pull request by kenmays**
Closes: #6602

### CODE REVIEW

File: app-office/libreoffice/patches/libreoffice-6.4.7.2.patchset [hunk 1]
```diff
@@ -429,5 +429,876 @@ index 712a61d..2c3d5f1 100644
...
+-#endif
++	(::std::ostringstream() << stream).str()
+ 
+ /**
+     @page sal_log Basic logging functionality.
+-- 
+2.30.2
+
+
+From 35e272a5c21f7b1b36b58b3693b5d880cb22a6ee Mon Sep 17 00:00:00 2001
```
Comment (Begasus) [inline app-office/libreoffice/patches/libreoffice-6.4.7.2.patchset:521]: Is this needed? (already included in the orcus patchset)

File: app-office/libreoffice/patches/libreoffice-6.4.7.2.patchset [hunk 2]
```diff
@@ -429,5 +429,876 @@ index 712a61d..2c3d5f1 100644
...
++#if defined _LIBCPP_VERSION \
++	|| (defined _GLIBCXX_RELEASE >= 12 || (_GLIBCXX_RELEASE == 11 && __GLIBCXX__ > 20210428))) \
++	|| (defined _MSC_VER && _MSC_VER >= 1915)
+ #define SAL_STREAM(stream) \
+-    (::std::ostringstream() << stream).str()
+-#else
+-#define SAL_STREAM(stream) \
+-    (dynamic_cast< ::std::ostringstream & >(::std::ostringstream() << stream).str())
+-#endif
++	(::std::ostringstream() << stream).str()
```
Comment (Begasus) [inline app-office/libreoffice/patches/libreoffice-6.4.7.2.patchset:513]: Shouldn't there be an `#endif` here?

**Comment by Begasus** [comment]
Build still fails here ...

File: app-office/libreoffice/patches/libreoffice-6.4.7.2.patchset (hunk 1 above)
Comment (kenmays) [reply app-office/libreoffice/patches/libreoffice-6.4.7.2.patchset]: Fixed

**Comment by kenmays** [comment]
Refreshed recipe.

**Comment by Begasus** [comment]
Build has been running for almost 2 days, previous build error occurred way sooner before, follow up on buildmaster @kenmays

**Pull request merged**